README.md

```diff
@@ -114,22 +114,76 @@ stages:
     response:
       save:
         $ext:
-          # Saving using this extension function allows us to save values into nested objects.
-          function: seekret.apitest:dots_save
+          function: seekret.apitest:save_authorization
           extra_kwargs:
-            headers:
-              # This will take the X-Auth-Token header from the response and put it in the header
-              # of future test stages.
-              seekret-runtime.v1.auth.headers.X-Auth-Token: '"X-Auth-Token"'
+            # This will take the X-Auth-Token header from the response and put it in the header
+            # of future test stages.
+            headers: '"X-Auth-Token"'
+            type: header
+            data:
+              target_header: X-Auth-Token
 ```
 
 Before running a tavern test, the Seekret library will prepend the custom authentication stage to the test, so it
 executes prior to the test stages defined in the test file.
 
-The stage is in your full control, but you must save the authorization values using the `seekret.apitest:dots_save`
-extension function. The `dots_save` extension function allows saving response values into nested objects.
+The stage is in your full control, but you must save the authorization values using
+the `seekret.apitest:save_authorization`
+extension function.
+
+### `seekret.apitest:save_authorization`
+
+`save_authorization` is a tavern extension function, intended to use in authentication requests. The function saves the
+authorization tokens so that later stages can use the authorization values.
+
+You can use the following "kwargs" to specify the wanted behaviour:
+
+| Argument Name | Description                                                                                             |
+|---------------|---------------------------------------------------------------------------------------------------------|
+| `type`        | Type of authorization method (currently supported: `header`, `bearer`).                                 |
+| `data`        | Additional data for the authorization method. See the next table for expected values.                   |
+| `headers`     | JMES path to the response header to take the authorization token from. Can't be used with `json`.       |
+| `json`        | JMES path to the authorization token in the JSON content of the response. Can't be used with `headers`. |
+
+| `type` value | `data` fields                                                              |
+|--------------|----------------------------------------------------------------------------|
+| `header`     | `target_header` - the name of the header to set to the authorization token |
+| `bearer`     | N/A                                                                        |
```

Comment:
porque bearer is not available?

Reply:
oh i saw in the example. clarify in this table that `data` is not required

Reply:
Fixed